timesketch/lib/datastores/neo4j.py

```diff
@@ -16,34 +16,46 @@
 from neo4jrestclient.client import GraphDatabase
 from neo4jrestclient.constants import DATA_GRAPH
 
+HIDDEN_FIELDS = [
+    'type', 'id', 'sketch_id', 'source', 'target', 'es_index_name', 'es_query',
+]
+
 # Schema for Neo4j nodes and edges
 SCHEMA = {
     u'nodes': {
         u'WindowsMachine': {
-            u'label_template': u'\uf009  {hostname}',
+            u'label': u'\uf009\u00A0\u00A0{hostname}',
```

Comment:
Maybe use uf17a (the Windows logo) instead?

Reply:
No. Windows logo does not look good when in bold font.